test/dotenv-flow.spec.js

```diff
@@ -264,6 +264,39 @@ describe('dotenv-flow', () => {
     });
   });
 
+  describe('when the `node_env` option is provided', () => {
+    const directory = getFixtureProjectPath('node-env-local');
+
+    it('uses that value to load specific `.env.*` files with a different `NODE_ENV` value set', async () => {
+      let environment, variables;
+
+      variables = await execHelper('print-env-with-node_env.js', directory);
+
+      expect(variables).to.include({
+        DEFAULT_ENV_VAR: "ok",
+        TEST_ENV_VAR: "ok",
+        TEST_LOCAL_VAR: "ok"
+      });
+
+      environment = {
+        NODE_ENV: 'production'
```

Comment:
And provide these custom env from here:
```suggestion
        NODE_ENV: 'production',
        CUSTOM_ENV: 'test'
```

Reply:
Ah, got it. I'll do that. Currently I am passing the test var in `print-env-with-node_env.js` like so
```
require('../../lib/dotenv-flow').config({
  node_env: 'development'
});
```

Reply:
With NODE_ENV set in the spec file